src/month.js

```diff
@@ -145,6 +171,7 @@ CalendarMonth.propTypes = {
 	dayAbbrevs: PropTypes.arrayOf(PropTypes.string).isRequired,
 	dayHeaderClassName: PropTypes.string,
 	firstDay: PropTypes.instanceOf(moment).isRequired,
+  gutterWidth: PropTypes.string,
```

Comment:
Eh, everything around it is tabs, and I want it to be spaces. I'm gonna punt on whitespace until I get the linting PR up.